### PR TITLE
Add regex for some porn sites (category-porn)

### PR DESCRIPTION
**Pull request by Carrot3628**

### CODE REVIEW

**Review by Amukev — APPROVED** [review]

**Review by rootmelo92118 — CHANGES_REQUESTED** [review]
sort it. and please submit the source.

**Review by Amukev — APPROVED** [review]

**Comment by rootmelo92118** [comment]
@Amukev The contributor does not sort the regular expression which had been submitted.

**Review by Amukev — APPROVED** [review]

File: data/category-porn [hunk 1]
```diff
@@ -6094,3 +6094,6 @@ regexp:(^|\.)zh9[0-7]\.net$
 regexp:(^|\.)zy[0-9]{4}\.top$
 regexp:(^|\.)zztt[1-6][0-9]\.com$
 regexp:(^|\.)zzz[1-9][0-9]\.top$
+regexp:(^|\.)x9av\d+\.com$
```
Comment (Amukev) [inline data/category-porn:6097]:
```suggestion
regexp:(^|\.)x9av\d+\.com$
```

**Review by Amukev — CHANGES_REQUESTED** [review]

File: data/category-porn (hunk 1 above)
Comment (Amukev) [reply data/category-porn:6097]: []()

Comment (Amukev) [reply data/category-porn:6097]: []()

**Pull request closed without merging**